src/sidebar/services/test/annotations-test.js

```diff
@@ -35,33 +35,51 @@ describe('annotationService', () => {
   let fakeStore;
   let fakeApi;
   let fakeAnnotationMapper;
-  let fakeIsNew;
   let fakeStreamer;
   let fakeStreamFilter;
 
   let fakeUris;
   let fakeGroupId;
 
+  let fakeIsNew;
+  let fakeIsPublic;
+  let fakePrivatePermissions;
+  let fakeSharedPermissions;
+
   beforeEach(() => {
     sinon.stub(console, 'error');
     searchClients = [];
     longRunningSearchClient = false;
+
     fakeAnnotationMapper = {
       loadAnnotations: sinon.stub(),
       unloadAnnotations: sinon.stub(),
     };
+
     fakeApi = {
       annotation: {
         create: sinon.stub().resolves(fixtures.defaultAnnotation()),
         update: sinon.stub().resolves(fixtures.defaultAnnotation()),
       },
       search: sinon.stub(),
     };
+
     fakeIsNew = sinon.stub().returns(true);
+    fakeIsPublic = sinon.stub().returns(true);
+
+    fakePrivatePermissions = sinon.stub().returns({
+      read: ['acct:foo@bar.com'],
+      update: ['acct:foo@bar.com'],
+      delete: ['acct:foo@bar.com'],
+    });
+    fakeSharedPermissions = sinon.stub().returns({
+      read: ['group:__world__'],
+    });
```

Comment:
Hmm, I have some misgivings about mocking simple + pure utility functions like `isNew`, `isPublic`, `privatePermissions` and `sharedPermissions` as it adds friction when refactoring (more mocks to update, more risk of an integration failure).

This is just me musing aloud - please leave this code as-is in this PR.

Reply:
Worthy of a longer conversation. I have had situations in the past week in which I've found myself going back and mocking util functions I hadn't mocked because it suited the unit tests better. And I want to avoid the situation in which we're hiding implementation details in fixtures or generally over-relying on fixtures, which add a layer of abstraction that I feel can be problematic in some places. To wit, if I've got tests in a component that are dependent on the specific value returned by a given dependency, I'm going to want to mock it. If the tests aren't impacted by a util function's variances, I might not choose to mock it. It's a question of communication to developers within the tests first and foremost: a line like:

`fakeMyMetadataUtil.thisAnnotationHasAFlower.returns(true);`

at the start of a test has a stronger signal than floweriness being buried in a fixture somewhere. So I'd like to continue using that pattern as I go.

The particular case you called out here I think may be a code smell of the way we're handling permissions as much as anything. Which we know needs scrutiny. 

Overall, I fall on the "mock the hell out of it" side of the argument, but this is just code, my mind is open :)